stories/black-belt-climate-ej.stories.mdx

```diff
@@ -259,18 +228,19 @@ taxonomy:
 </Block>
 
 
-<Block style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', margin: '0 auto', width: '80%' }}>
-  <Prose style={{ textAlign: 'justify', display: 'flex', alignItems: 'center' }}>
-    <p>
-     Additionally, outdoor occupations such as transportation and construction have become crucial sources of income and have shown growth over the past few years. Data from the U.S. Census Bureau, which examines the number of establishments by employment category in counties like Choctaw and Butler, underscores the economic importance of these sectors.
-    </p>
-  </Prose>
+<Block style={{justifyContent: 'center', alignItems: 'center', margin: '0 auto', width: '80%' }}>
   <Figure style={{ flex: '0 0 auto'}}>
     <Image
       src={new URL('./BB_Estab_AL.png', import.meta.url).href}
       style={{ maxWidth: '800px', height: 'auto' }}
     />
```

Comment:
these images are blurry, maybe use the [`<Chart>` component](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/MDX_BLOCKS.md#chart)?